Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -329,17 +329,6 @@ static uint32_t convertSystemLayoutDirection(NSUserInterfaceLayoutDirection dire
 
 #endif // PLATFORM(MAC)
 
-#if PLATFORM(IOS_FAMILY)
-static void hardwareKeyboardAvailabilityChangedCallback(CFNotificationCenterRef, void* observer, CFStringRef, const void*, CFDictionaryRef)
-{
-    ASSERT(observer);
-    RetainPtr webView { (__bridge WKWebView *)observer };
-    if (!webView)
-        return;
-    webView->_page->hardwareKeyboardAvailabilityChanged();
```

Comment:
I think this patch introduces a subtle behavior change due to the removal of this call.

It seems that nothing calls `-[WKContentView _hardwareKeyboardAvailabilityChanged]` anymore, which is responsible for reloading input views.

Reply:
Ah, good point, will address this in the upcoming patch.

Thanks for reviewing!